docs/release-strategy.md

```diff
@@ -47,19 +47,26 @@ The Release Manager for each release is identified in the Description of the Mil
 
 The following are the steps for how Y-stream and Z-stream releases gets cut.
 
-1. Maintainers determine a commit on the main branch that will serve as the basis for the next release.
-    - For a Z-stream release skip this step.
-2. For a Y-Stream release: create a new branch in the format `release-vX.Y`.
-    - For a Z-Stream release skip this step.
-3. For a Z-Stream release: backport all relevant commits from `main` to the `release-X.Y` branch.
-    - For a Y-Stream release skip this step.
-4. Create a new release on GitHub. The following is automated:
-    - Tagging the branch on GitHub
-    - Creating a change log on GitHub
-    - Version number of the Python package is derived from the tag name
-5. Validate the release (manual testing).
-6. Move the `stable` tag to the newest release.
-7. Announce release on the following channels:
-    - InstructLab Slack
-8. Create a milestone on GitHub for the next release without a milestone.
-    - For a Z-Stream release skip this step.
+### Y-Stream
+
+1. Determine a commit on the main branch that will serve as the basis for the next release - most of the time this should be the latest commit.
+1. Create a new release branch in the format `release-vX.Y` - this should match `main` when created.
+1. Validate the release branch with an E2E test.
+1. Create a new release on GitHub targeting the release branch and using the latest Y-Stream tag as the previous release (e.g. `0.15.1` preceeds `0.16.0`).
+1. Move the `stable` tag to the new release (note this tag is set to be deprecated on September 1st, 2024 - users should use PyPi to install the latest "stable" release)
+1. Announce release on the following channels on Slack:
+    - #announce
+    - #frontend
+    - #users
+1. Create a milestone on GitHub for the next release without a milestone.
+
+### Z-Stream
+
+1. Backport all relevant commits from `main` to the `release-X.Y` branch - this can be done automatically with Mergify or manually if preferred.
```

Comment:
```suggestion
1. Backport all relevant commits from `main` to the `release-X.Y` branch - this can be done automatically with Mergify or manually if preferred. A backport using Mergify is done by adding a comment to the PR with the change merged to `main` with the contents `@mergifyio backport <branch-name>`.
```